src/components/FolderToolBar/FolderToolBar.jsx

```diff
@@ -0,0 +1,58 @@
+import FolderButton from "../FolderButton/FolderButton";
+import AddFolderButton from "../AddFolderButton/AddFolderButton";
+import ShareEditDeleteButton from "../ShareEditDeleteButton/ShareEditDeleteButton";
+import "./FolderToolBar.css";
+
+const BUTTONS = [
+  {
+    icon: "images/share.svg",
+    text: "공유",
+  },
+  {
+    icon: "images/pen.svg",
+    text: "이름 변경",
+  },
+  {
+    icon: "images/trash.svg",
+    text: "삭제",
+  },
+];
+
+const FolderToolBar = ({ folderLists, chosenFolderId, onClickFolder }) => {
```

Comment:
P3:
전체 버튼인지 아닌지 여부가 코드내에서 반복되니 해당 값을 변수로 빼면 좋겠네요.

```suggestion
const FolderToolBar = ({ folderLists, chosenFolderId, onClickFolder }) => {
  const isAllFolderButton = chosenFolderId === "all"
```